### ISSUE TITLE
TMDb attribution

### ISSUE THREAD

**Issue by kevnfu** [kevnfu/e-unibus-pluram#9]
On the TMDb [API FAQ](https://www.themoviedb.org/faq/api)

> What are the attribution requirements?
> 
> You shall use the TMDb logo to identify your use of the TMDb APIs. You shall place the following notice prominently on your application: "This product uses the TMDb API but is not endorsed or certified by TMDb." Any use of the TMDb logo in your application shall be less prominent than the logo or mark that primarily describes the application and your use of the TMDb logo shall not imply any endorsement by TMDb. When attributing TMDb, the attribution must be within your application's "About" or "Credits" type section.
> 
> When using a TMDb logo, we require you to use one of our approved images.

I believe that this means it _only_ has to be in the "About" or "Credits" section. A list of the logos can be found [here](https://www.themoviedb.org/about/logos-attribution?language=en). So that's a page we need.

Also I want to be able to directly link to the Series page on TMDb's site. The data is all generated by people manually entering data so this will make it easier if someone notices that a show hasn't been updated in a while. Where would be put that?

**Comment by paintgoo** [comment]
Did you write this section? Assuming yes.
_I believe that this means it only has to be in the "About" or "Credits" section. A list of the logos can be found here. So that's a page we need._

Yes, maybe we add **About** to the NavBar. Do we have to both use their logo as well as a written out attribution? Would rather just the later.

Maybe if you click on the title on your list it links out to the series page. Might also be useful to click to it from the search as well. So after searching for a show you can add it to your list or go find out more info about it.

Linking out to someone else's site is kosher?

**Comment by paintgoo** [comment]
Also, how would you link back? Are we talking about using the browser's back button or opening in new tabs?

**Comment by kevnfu** [comment]
Yeah sorry, it was missed a newline. I believe their terms of service require both the logo and the disclaimer.

For getting info on a series I would just want to pop up a [modal](http://getbootstrap.com/javascript/#modals), so the user doesn't have to leave the page. What do you think?

Maybe on the modal there will be a link to the [tmdb](https://www.themoviedb.org/tv/56570-outlander?language=en) page?  Of course linking to someone's page is good. Why would they make a website where they don't want people to visit?
